centreon-broker/grpc/test/grpc_test_keys/ca_1234.crt

```diff
@@ -0,0 +1,33 @@
+-----BEGIN CERTIFICATE-----
+MIIFpTCCA42gAwIBAgIUPvltjGPvcLx6j5/UlpoG70lUbZ4wDQYJKoZIhvcNAQEL
+BQAwYTELMAkGA1UEBhMCU1AxDzANBgNVBAgMBkZyYW5jZTERMA8GA1UEBwwIVG91
+bG91c2UxDTALBgNVBAoMBFRlc3QxDTALBgNVBAsMBFRlc3QxEDAOBgNVBAMMB1Jv
+b3QgQ0EwIBcNMjIwMzE1MTUxMzMwWhgPMjEyMjAyMTkxNTEzMzBaMGExCzAJBgNV
+BAYTAlNQMQ8wDQYDVQQIDAZGcmFuY2UxETAPBgNVBAcMCFRvdWxvdXNlMQ0wCwYD
```

Comment:
Do you really need these certificates? Is it not possible to generate during the tests?